.github/workflows/buildRelease.yml

```diff
@@ -67,6 +67,24 @@ jobs:
       with:
         release_id: ${{ github.event.inputs.release_id }}
         assets_path: ./build_tools/python_deploy/wheelhouse/torch*.whl
+
+    - name: Download nightly pytorch and torchvision
+      if: github.event.inputs.release_id != ''
+      run: |
+        mkdir deps
```

Comment:
Where is this `deps` folder created? How is this relating to the URL under which we can later on find the wheel?

Reply:
It's created in the working directory of the github actions run, which I think is the same directory as the git working directory. This does not affect the URL of where the wheel is uploaded to afaik.